content/en/blog/_posts/2024-10-28-memory-manager-moves-to-ga/index.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: blog
+title: "Memory Manager goes GA"
+date: 2024-10-28
+slug: memory-manager-goes-ga
+author: >
+  [Talor Itzhak](https://github.com/Tal-or) (Red Hat)
+---
+
+With Kubernetes 1.32, the Memory Manager has officially graduated to General Availability (GA),
+marking a significant milestone in the journey toward efficient and predictable memory allocation for containerized applications.
+Since Kubernetes v1.22, where it graduated to beta, the Memory Manager has proved itself reliable, stable and a good complementary feature for the [CPU Manager](https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/).
```

Comment:
```suggestion
Since Kubernetes v1.22, where it graduated to beta, the Memory Manager has proved itself reliable, stable and a good complementary feature for the
[CPU Manager](/docs/tasks/administer-cluster/cpu-management-policies/).
```